automated/linux/ltp-open-posix/ltp-open-posix.sh

```diff
@@ -63,6 +66,23 @@ parse_ltp_output() {
 
 }
 
+# Parse LTP open posix grp output
+parse_ltp_grp_output() {
+    sed -i.bak '/^PASS\|^FAIL\|^Testing/!d' "${GRP_OUTPUT_FILE}"
+    paste -d " "  - - - < "${GRP_OUTPUT_FILE}" | tee "${GRP_RESULT_FILE}"
+    sed -i 's/\s\s*/ /g' "${GRP_RESULT_FILE}"
+    sed -i 's/Testing //g' "${GRP_RESULT_FILE}"
+    {
+    grep -E "FAIL 0" "${GRP_RESULT_FILE}" | awk '{print $1 " " $2}' | sed 's/PASS/pass/; s/FAIL/fail/; s/CONF/skip/';
+    grep -v "FAIL 0" "${GRP_RESULT_FILE}" | awk '{print $1 " " $4}' | sed 's/PASS/pass/; s/FAIL/fail/; s/CONF/skip/';
+    grep -e "UNRESOLVED" -e "UNTESTED" -e "UNSUPPORTED" -e "HUNG" -e "SKIPPED" "${GRP_OUTPUT_FILE}.bak" \
+	    | awk -F':' '{print $1 $3}' \
+	    | awk -F'/' '{print $NF}' \
+	    | sed 's/SKIPPED/skip/; s/UNSUPPORTED/fail/; s/UNTESTED/fail/; s/UNRESOLVED/fail/; s/HUNG/skip/'
+    } >> "${RESULT_FILE}"
```

Comment:
Please share the latest LAVA test log links with this later code.

Reply:
ok, here is the LAVA job with these changes: https://lava.collabora.co.uk/scheduler/job/4189498